desktop-widgets/filterwidget2.ui

```diff
@@ -35,109 +35,95 @@
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
-       <item row="3" column="2">
-        <widget class="QDoubleSpinBox" name="minWaterTemp"/>
-       </item>
-       <item row="8" column="0">
-        <widget class="QLabel" name="label_7">
-         <property name="text">
-          <string>Tags</string>
-         </property>
+       <item row="13" column="2">
```

Comment:
Off-topic: I have been ridiculed for creating my UIs with normal C++ function calls instead of these XML-files. At least it gives reasonable and mergeable diffs, compared to this mess!